imdb-ratings/Controllers/GenreController.cs

```diff
@@ -14,26 +13,22 @@ namespace imdb_ratings.Controllers
     public class GenreController : ControllerBase
     {
         // GET api/<GenreController>/5
-        [HttpGet("{language}")]
+        [HttpGet("movies/genres")]
         public List<string> Get(string language)
```

Comment:
```suggestion
        public IEnumerable<string> Get(string language)
```
Need not have a concrete return. Can have generic